Makefile

```diff
@@ -21,7 +21,15 @@ install:
 # Testing and linting
 .PHONY: test
 test:
-	poetry run pytest metricflow/test/
+	poetry run pytest metricflow/test/ --ignore metricflow/test/model/dbt_cloud_parsing
+
+.PHONY: test-dbt
+test-dbt:
+	poetry run pytest metricflow/test/model/dbt_cloud_parsing
+
+.PHONY: test-all
+test-all:
```

Comment:
Unfortunate that we have to have a `test` and a `test-all`, but also for the time being I don't know what the alternative would be 😞

Reply:
I didn't even know this was needed until my `make test` blew up. I'm not sure how you would've even noticed given that you had the dbt packages installed for your dev work.

I think the longer term solution will involve re-packaging somehow, either making the dbt bits required as part of metricflow core library, or else having the dbt bits packaged as an extension that imports the metricflow core library. But that's quite a lot of work so I figured we can use this as a stopgap.

One smaller scale solution would be to have the dbt tests in their own test package, and then we could have test and test-dbt.